open-source.adoc

```diff
@@ -424,14 +422,14 @@ more people working on it, as the price has gone up, but it’s
 definitely sublinear.
 ____
 
-The 2x figure should be taken with a grain of salt, it's probably
-more, but his point is that the amount of development work scales
+The 2x figure should be taken with a grain of salt (it's probably
+more), but his point is that the amount of development labor has scaled
 sublinearly with the price of bitcoin. He's not sure whether that's a
-good thing or a bad thing. On one hand we'd get even better systems
+good or a bad thing. On the one hand we'd get even better systems
 with more funding, but on the other hand we'd get almost as good a
-system with less funding. The https://coredev.tech/[coredev.tech] he
-mentions is an initiative by developers to meet in person to discuss
-proposals, review code, and meet collaborators in person.
+system with less funding [comm: then more funding would only be a good thing, wouldn't it?]. The https://coredev.tech/[coredev.tech] he
```

Comment:
Yes, but I think his point is that each extra dollar in funding provides less and less value. The 1001th dollar is more valuable than the 1000001th. You get less and less improvement to Bitcoin for each extra dollar.

Reply:
I understand the concept of decreasing marginal utility in this context, it's the sentence "He's not sure whether that's a good or a bad thing" that confuses me. I would say that more funds are only a good thing, marginally decreasing in their goodness yes, but still a good thing. Did I misintepret the meaning?

Reply:
Oh lord, I'm not sure what to make of this text anymore. Thanks for pushing on this.

Thinking out loud below:

I now think he's talking about sublinear development cost (measured by number of people working on Bitcoin) with respect to BTC price. The price 100x'd but cost 2x'd. This means *more* bang for the buck for every extra buck. (bang = price increase). I think his argument is flawed because bang should be measured in development output (code improvements, review, new features, etc), not price. Price is a function of so much more than development work.

Should we ditch this part? I think it's borderline gibberish.

Maybe @adamjonas can chime in here? Background: We're discussing what Tadje Dryja is really saying in his talk, and if my text should be removed or improved: https://github.com/bitcoin-dev-philosophy/btcphilosophy/pull/77/files#diff-fbea55efe44fc7b7dd90ef8c6bd09e60c6f67d84562be51cdaf1d4cdaaedf6a2R408

Reply:
Yeah, let's ditch it. It is really confusing. let's remove from "If a talk about open souce development funding...".